.github/workflows/container-tests.yml

```diff
@@ -64,6 +65,6 @@ jobs:
           git_ref: ${{ matrix.branch }}
           tf_scope: ${{ matrix.tf_scope }}
           tmt_plan_regex: ${{ matrix.tmt_plan }}
-          pull_request_status_name: ${{ matrix.context }}
-          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ github.event.issue.number }};OS=${{ matrix.os_test }};TEST_NAME=test"
+          pull_request_status_name: "${{ matrix.context }}-${{ matrix.container-to-test }}
+          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ github.event.issue.number }};OS=${{ matrix.os_test }};TEST_NAME=test;TESTED_IMAGE=${{ matrix.container-to-test }}"
```

Comment:
As I am looking into the TMT plans, they currently do not support `TESTED_IMAGE`. How this variable will be used?

Reply:
variables are exported by TMT plans as `export TESTED_IMAGE`.

E.g. REPO_NAME is mentioned here: https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c9s.fmf#L10 and here: https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c9s.fmf#L21